docs/guides/functions.ipynb

```diff
@@ -440,7 +414,49 @@
   },
   {
    "cell_type": "markdown",
-   "id": "9137071a-e39c-4b66-afbd-697db38cc670",
+   "id": "af7fa72a-f272-43ff-800d-5ae37dd42b2b",
+   "metadata": {},
+   "source": [
+    "To retrieve a job's job_id by its position in the list, use `catalog.get_job_by_id()`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "id": "18d87f07-8a8e-4b93-80d0-7c07e49272e2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get the job_id for the job at index position 1\n",
+    "latest_job = old_jobs[1]\n",
+    "job_by_id = catalog.get_job_by_id(latest_job.job_id)\n",
+    "\n",
+    "# verify that the job ID is the same using both retrieval methods\n",
+    "assert job_by_id.job_id == latest_job.job_id"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "id": "913d1999-c5b5-426f-a5c5-954c41af8bfb",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<Job | 8a21f700-b42f-4676-846d-4d4ce37da987>\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Print the job_id for the job at index position 1\n",
+    "print(latest_job)"
```

Comment:
This can be part of the same code block that you had above. There's no need to split them out and it may be more confusing to split them. The code is tightly "coupled"